README.md

```diff
@@ -1,169 +1,112 @@
 # kube-spawn
 
-## Introduction
-
-__kube-spawn__ is a tool for creating a multi-node Kubernetes cluster
+`kube-spawn` is a tool for creating a multi-node Kubernetes cluster
 on a single machine, created mostly for developers __of__ Kubernetes.
 
 It aims to be as similar to the solutions recommended for production
 clusters as possible.
 
-## Getting started
+## Requirements
 
-### Build proper systemd-nspawn version
+* **Host:**
+  - `systemd-nspawn` at least version 233
+  - tested: Fedora 26, ...
```

Comment:
Can you drop this line? I guess you meant to say something like "kube-spawn has been tested on Fedora 26." which I would rather put into the intro above `## Requirements`, if needed at all.